docs/en/Microsoft.Caboodle/Preferences.xml

```diff
@@ -10,8 +10,21 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>A class to interact with the preferences/settings of the native platform.</summary>
+    <remarks>
+      <para>Each platform uses the platform provided native API’s for storing application/user preferences:</para>
+      <list type="bullet">
```

Comment:
These "bullets", is this an mdoc thing? I never actually did anything fancy for my docs, so I don't know.

Reply:
I'm so fancy.

DocWriter will help you insert these too FYI.